src/main/java/com/mentalhealthapi/mentalhealthapi/MentalHealthController.java

```diff
@@ -24,7 +24,7 @@ public String index() {
     }
 
     /**
-     * Fetch a list of all disorders.
+     * Fetch a list of all disorders
```

Comment:
One comment ended with a period while all others did not, so removing it would improve overall cleanness of comment. This is more inline with the comment standards of Baeldung. https://www.baeldung.com/java-clean-code#8-code-comments

Reply:
Thank you for your suggestion!